quickstarts/flutter-android/config.yml

```diff
@@ -0,0 +1,45 @@
+slug: flutter-android
+description: |
+  ## Comprehensive monitoring quickstart for Flutter Android
+  
+  Monitor your Flutter applications and analyze performance bottlenecks while operating in a live development environment. You can use this dashboard to visualize your web user experience with your application rendition.
+  
+  ## Why monitor Flutter Android?
+  Monitoring app performance helps you to take a preventative approach to enhance quality rather than waiting for a crash, issue, or user complaint to occur. The Newrelic’s Android and  Flutter agent allows you to track important metrics including network failures and http response, slow launches, tracking custom traces, and more.
+
+  By prioritizing active issues and marking errors as corrected, you may optimize the process once you have a comprehensive picture of all releases. Identify which release the problem first appeared in, merge duplicates, and assess whether it may get worse in the future. Add commit information to send deploy emails instantly and suggest an owner for each application error.
+  
+  ## What’s included in this quickstart?
+  New Relic's Android and Flutter monitoring agent captured the metrics out of the box:
+  - Be alerted on critical issues affecting your Flutter Android application.
+  - Dashboards (error rate, app launches, app installations, app crashes and more).
```

Comment:
can be updated to: `  - Dashboards (error rate, app launches, app installations, app crashes, etc.).`

Reply:
Usage of Full stop (.) not required for bullet points.

Reply:
I have updated it as per your comments.
Thank you...